packages/perseus/src/widgets/expression.tsx

```diff
@@ -574,59 +602,81 @@ export class Expression extends React.Component<Props, ExpressionState> {
-                    opened={this.state.showErrorTooltip}
-                    title={ERROR_TITLE}
-                    content={ERROR_MESSAGE}
-                >
-                    <MathInput
-                        // eslint-disable-next-line react/no-string-refs
-                        ref="input"
-                        className={ApiClassNames.INTERACTIVE}
-                        value={this.props.value}
-                        onChange={this.changeAndTrack}
-                        convertDotToTimes={this.props.times}
-                        buttonSets={this.props.buttonSets}
-                        onFocus={this._handleFocus}
-                        onBlur={this._handleBlur}
-                        hasError={this.state.showErrorStyle}
-                        extraKeys={this.props.keypadConfiguration?.extraKeys}
-                        analytics={
-                            this.props.analytics ?? {
-                                onAnalyticsEvent: async () => {},
+                    <View style={a11y.srOnly} role="alert">
+                        {this.state.showErrorTooltip &&
+                            ERROR_TITLE + " " + ERROR_MESSAGE}
+                    </View>
+                    <Tooltip
+                        forceAnchorFocusivity={false}
+                        opened={this.state.showErrorTooltip}
+                        title={ERROR_TITLE}
+                        content={ERROR_MESSAGE}
+                    >
+                        <MathInput
+                            // eslint-disable-next-line react/no-string-refs
+                            ref="input"
+                            className={ApiClassNames.INTERACTIVE}
+                            value={this.props.value}
+                            onChange={this.changeAndTrack}
+                            convertDotToTimes={this.props.times}
+                            buttonSets={this.props.buttonSets}
+                            onFocus={this._handleFocus}
+                            onBlur={this._handleBlur}
+                            hasError={this.state.showErrorStyle}
+                            ariaLabel={
+                                this.props.ariaLabel ||
+                                this.context.strings.mathInputBox
                             }
-                        }
-                    />
-                </Tooltip>
-            </div>
+                            extraKeys={
+                                this.props.keypadConfiguration?.extraKeys
+                            }
+                            analytics={
+                                this.props.analytics ?? {
+                                    onAnalyticsEvent: async () => {},
+                                }
+                            }
+                        />
+                    </Tooltip>
+                </div>
+            </View>
         );
     }
 }
 
+const styles = StyleSheet.create({
```

Comment:
Did Caitlyn get a chance to take a look at these padding values for mobile and desktop?